services/api/src/typeDefs.js

```diff
@@ -579,6 +579,7 @@ const typeDefs = gql`
     contentType: String
     notificationSeverityThreshold: ProblemSeverityRating
     organization: Int
+    pid: Int
```

Comment:
What is `pid` used for here? This query should only return top level notifications, not anything related to projects?

Reply:
Added that so we can show both project assigned and unassigned notifications. Happy to remove it though if we don't want to go that deep.

Reply:
Yeah, I think for this particular one it should be omitted.

We can revisit it later if required. But a `projectsbynotification` resolver would probably be a good fit here instead.